com.ibm.wala.core/src/main/java/com/ibm/wala/ipa/callgraph/impl/Util.java

```diff
@@ -119,6 +123,47 @@ public static void addBypassLogic(
     }
   }
 
+  /**
+   * Removes some overhead from the previous method, removes the need to pass in an AnalysisScope
+   * parameter
+   */
```

Comment:
You should copy-paste the relevant documentation from the old method rather than writing this